packaging/torchvision/meta.yaml

```diff
@@ -39,6 +39,7 @@ build:
     - FORCE_CUDA
     - NVCC_FLAGS
```

Comment:
Maybe we can remove this flag now?

Reply:
I'm not sure we can remove it, as `NVCC_FLAGS` is still being used in the windows files:

https://github.com/pytorch/vision/blob/978ba613518dd5b8d04c2a717e6e5ccf7fb172c3/packaging/windows/cuda101.bat#L35-L40

Reply:
I wonder if we still need this given that PyTorch already set those in [cpp_extensions](https://github.com/peterjc123/pytorch/blob/65087dd1d4c1e503281cd676e5563e9e64b2a926/torch/utils/cpp_extension.py#L178-L183).

Maybe @peterjc123 could chime in?

Reply:
I removed it in 5055fa8 and it looks like the CI is still happy

Reply:
Yeah but that codepath might not be exercised on the PR CI, but only on the release CIs.

I'd love to get @peterjc123 to double-check this part before we merge this

Reply:
You should do it in the script as well. `__CUDA_NO_HALF_OPERATORS__ ` and `-expt-relaxed-constexpr` are set in `torch/utils/cpp_extension.py` so I think it should be okay.

Reply:
Great, thanks for the confirmation @peterjc123 !

I've opened https://github.com/pytorch/vision/issues/3617 to track removing this file